src/OSPSuite.Core/Domain/Builder/ExpressionParameter.cs

```diff
@@ -2,5 +2,14 @@
 {
    public class ExpressionParameter : PathAndValueEntity
    {
+      public bool IsGlobalRelativeExpression()
```

Comment:
ok this is weird lol. A property that looks like a. method that calls an extension method?

Reply:
or is this because it's not a parameter maybe?